docs/CONTRIBUTING.md

```diff
@@ -94,30 +94,20 @@ To run the Vale prose linter locally, `brew install vale` and `vale docs/`.
 
 ## Governance
 
-ViewComponent is built by over a hundred members of the community. Project membership has three levels:
-
-### Triage
-
-ViewComponent triagers are able to manage issues and pull request by assigning owners and labels, closing issues and marking duplicates.
-
-After helping with the project by participating in issues, pull requests, and/or discussions, members of the community are welcome to request triage access by opening a pull request to update this list:
-
-The triagers team is @andrewmcodes, @bbugh, @boardfish, @cesariouy, @dark-panda, @dylnclrk, @g13ydson, @horacio, @jcoyne, @johannesengl, @kaspermeyer, @mellowfish, @metade, @nashby, @rainerborene, @rdavid1099, @spdawson, @yhirano55, and @vinistock.
```

Comment:
Is the triage team being retired? I think if so, it'd still be good to encourage folks to get involved in issue triage even if there's no direct role attached to it. It was a good onramp for me to get involved in contributing to ViewComponent itself.

Reply:
That's a fair point. I removed it from the docs as none of the folks with `triage` have been active in that role. I'll re-add something 👍🏻